src/bindings/python/setup.cfg

```diff
@@ -78,8 +78,8 @@ show_column_numbers = True
 show_error_context = True
 show_absolute_path = True
 pretty = True
-follow_imports=normal
+follow_imports = normal
 disallow_untyped_defs = True
-disallow_untyped_calls = True
+disallow_untyped_calls = False
```

Comment:
Why did you change it?

Reply:
Due to raising Python version in GitHub workflows a higher version of `numpy` was installed, which caused `mypy` to have issues with it's untyped functions. It is one of the symptoms of recent weird flake8/mypy behaviour I talked about earlier - today the issues do not reproduce locally. I changed it back to `True`, but `mypy` finds errors in `numpy` functions:
`Call to untyped function "np.concatenate" in typed context`
It seems like the function is not typed, but it's a third party library. I'll see if I can solve it in a more elegant way than changing the config.

Reply:
I ignored type checking in 4 lines which were using problematic `numpy` functions instead of disabling the check altogether.